sdk/attestation/azure-security-attestation/azure/security/attestation/_models.py

```diff
@@ -69,242 +79,244 @@ class PolicyCertificatesModificationResult(object):
 
-class PolicyResult(object):
-    """ PolicyResult represents the result of a :meth:`azure.security.attestation.AttestationAdministrationClient.set_policy` 
-    or :meth:`azure.security.attestation.AttestationAdministrationClient.reset_policy`  API call.
 
-    The `PolicyResult` class is returned as the body of an attestation token from
+class AttestationPolicyResult(object):
+    """AttestationPolicyResult represents the result of a
+    :meth:`azure.security.attestation.AttestationAdministrationClient.set_policy`
+    or :meth:`azure.security.attestation.AttestationAdministrationClient.reset_policy`
+    API call.
+
+    The `AttestationPolicyResult` class is returned as the body of an attestation token from
     the attestation service. It can be used to ensure that the attestation service
     received the policy object sent from the client without alteration.
-    
-    :param policy_resolution: The result of the policy set or
-        reset call.
+
+    :param policy_resolution: The result of the policy set or reset call.
     :type policy_resolution: azure.security.attestation.PolicyModification
     :param policy_signer: If the call to `set_policy` or `reset_policy`
-        had a :class:`AttestationSigningKey` parameter, this will be the certificate
+        had a `signing_certificate` parameter, this will be the certificate
         which was specified in this parameter.
     :type policy_signer: azure.security.attestation.AttestationSigner
     :param str policy_token_hash: The hash of the complete JSON Web Signature
         presented to the `set_policy` or `reset_policy` API.
 
     """
+
     def __init__(self, policy_resolution, policy_signer, policy_token_hash):
-        #type:(PolicyModification, JSONWebKey, str) -> None
+        # type: (PolicyModification, JSONWebKey, str) -> None
         self.policy_resolution = policy_resolution
         self.policy_signer = AttestationSigner._from_generated(policy_signer)
         self.policy_token_hash = policy_token_hash
 
     @classmethod
     def _from_generated(cls, generated):
-        #type:(GeneratedPolicyResult)->PolicyResult
+        # type: (GeneratedPolicyResult, str) -> AttestationPolicyResult
+        # If we have a generated policy result or policy text, return that.
         if not generated:
-            return cls(None, None, None)
-        return cls(generated.policy_resolution, generated.policy_signer, generated.policy_token_hash)
+            return None
+        return AttestationPolicyResult(
+            generated.policy_resolution,
+            generated.policy_signer,
+            generated.policy_token_hash,
+        )
+
 
-class AttestationResult(object):
-    """ An AttestationResult represents the claims returned from the attestation
+class AttestationResult(object):  # pylint: disable=too-many-instance-attributes
+    """An AttestationResult represents the claims returned from the attestation
```

Comment:
Again, for consistency and brevity:
```suggestion
    """Represents the claims returned from the attestation
```